cfme/containers/image.py

```diff
@@ -203,24 +203,26 @@ def perform_smartstate_analysis_multiple_images(
         # task_name change from str to regular expression
         # the str compile on tasks module
         image_enities_names = []
+        delay = 5
+        silent_failure = True
```

Comment:
QUESTION: why are there hardcoded?

Reply:
No special reason, Just to understand your question, what is your suggestion?
Are you suggest to use this variable values directly on the function call?

Reply:
@brahmanim  delay is already defaulted to 5 - https://github.com/ManageIQ/integration_tests/pull/7337/files/6bc76133010fd1e4abcf5f2273c39d001e4639c8#diff-1fe1d4bbe572be5ebf426e918f68f31fR261
and `silent_failure` better to pass as attribute directly into `is_successfully_finished`. I don't see a need in additional local variables

Reply:
OK, just wanted to be sure before I make changes.

Reply:
Since i Will move back to WIP, Are you have any forther comments?

Reply:
nothing else ;) PR is fine. and even more - green PRT is here!